solvers.tex

```diff
@@ -60,7 +60,7 @@
 >>> solveset(x - x, x, domain=S.Reals)
 (-oo, oo)
 >>> solveset(x - x, x, domain=S.Complexes)
-S.Complexes
```

Comment:
I think why sstr was used was because of differences in Python 2 and 3 of repr.
Python3
```
In [9]: str(set([(8, 4, 4, 4, 0, 0)]))
Out[9]: '{(8, 4, 4, 4, 0, 0)}'

In [10]: sstr(set([(8, 4, 4, 4, 0, 0)]))
Out[10]: 'set([(8, 4, 4, 4, 0, 0)])'

In [11]: repr(set([(8, 4, 4, 4, 0, 0)]))
Out[11]: '{(8, 4, 4, 4, 0, 0)}'
```
Python2
```
In [5]: str(set([(8, 4, 4, 4, 0, 0)]))
Out[5]: 'set([(8, 4, 4, 4, 0, 0)])'

In [6]: sstr(set([(8, 4, 4, 4, 0, 0)]))
Out[6]: 'set([(8, 4, 4, 4, 0, 0)])'

In [7]: repr(set([(8, 4, 4, 4, 0, 0)]))
Out[7]: 'set([(8, 4, 4, 4, 0, 0)])'
```